### PR TITLE
label minikube nodes

### PR DESCRIPTION
**Pull request by medyagh**
After this PR minikube nodes will be annonated with a label, 
would help to identify this was created by minikube 
here are the labels after this PR:

```
medya@medya:~/workspace/minikube (annotate_node)$ kubectl get nodes --show-labels 
NAME   STATUS   ROLES    AGE     VERSION   LABELS
p1     Ready    master   4m53s   v1.17.3   

beta.kubernetes.io/arch=amd64,beta.kubernetes.io/os=linux,
k8s.minikube.io/commit=aa91f39ffbcf27dcbb93c4ff3f457c54e585cf4a-dirty,
k8s.minikube.io/name=p1,
k8s.minikube.io/updated_at=2020_02_20T12_05_35_0700,
k8s.minikube.io/version=v1.7.3,
kubernetes.io/arch=amd64,
kubernetes.io/hostname=p1,
kubernetes.io/os=linux,
node-role.kubernetes.io/master=
```

minikube added labels are : 

```
k8s.minikube.io/commit=aa91f39ffbcf27dcbb93c4ff3f457c54e585cf4a-dirty,
k8s.minikube.io/name=p1,
k8s.minikube.io/updated_at=2020_02_20T12_05_35_0700,
k8s.minikube.io/version=v1.7.3,


```

### CODE REVIEW

File: pkg/minikube/bootstrapper/kubeadm/kubeadm.go [hunk 1]
```diff
@@ -483,3 +488,30 @@ func (k *Bootstrapper) applyKicOverlay(cfg config.MachineConfig) error {
...
+	// /var/lib/minikube/binaries/v1.17.3/kubectl label nodes --kubeconfig /var/lib/minikube/kubeconfig   k8s.minikube.io/version=1.7.3 --all --overwrite
+	cmd := exec.Command("sudo",
+		path.Join(vmpath.GuestPersistentDir, "binaries", cfg.KubernetesConfig.KubernetesVersion, "kubectl"),
+		"label", "nodes", verLbl, commitLbl, nameLbl, createdAtLbl, "--all", "--overwrite",
+		fmt.Sprintf("--kubeconfig=%s", path.Join(vmpath.GuestPersistentDir, "kubeconfig")))
+
+	rr, err := k.c.RunCmd(cmd)
+	elapsed := time.Since(start)
+	if elapsed > (1 * time.Second) {
+		glog.Infof("Completed: %s: (%s)", rr.Command(), elapsed)
```
Comment (tstromberg) [inline pkg/minikube/bootstrapper/kubeadm/kubeadm.go]: Please remove: this log line is already included by command-runners.

File: pkg/minikube/bootstrapper/kubeadm/kubeadm.go [hunk 2]
```diff
@@ -483,3 +488,30 @@ func (k *Bootstrapper) applyKicOverlay(cfg config.MachineConfig) error {
 	}
 	return nil
 }
+
+// applyNodeLabels applies minikube labels to all the nodes
+func (k *Bootstrapper) applyNodeLabels(cfg config.MachineConfig) error {
+	start := time.Now()
+	// based on ISO 8601 (RFC 3339) except converting - and : to _ because of kubernetes label restriction
+	createdAtLbl := "k8s.minikube.io/updated_at=" + start.Format("2006_01_02T15_04_05_0700")
```
Comment (tstromberg) [inline pkg/minikube/bootstrapper/kubeadm/kubeadm.go]: this should probably be minikube.k8s.io

File: pkg/minikube/bootstrapper/kubeadm/kubeadm.go [hunk 3]
```diff
@@ -483,3 +488,30 @@ func (k *Bootstrapper) applyKicOverlay(cfg config.MachineConfig) error {
...
+	verLbl := "k8s.minikube.io/version=" + version.GetVersion()
+	commitLbl := "k8s.minikube.io/commit=" + version.GetGitCommitID()
```
Comment (tstromberg) [inline pkg/minikube/bootstrapper/kubeadm/kubeadm.go]: Can we omit the commit id? At least until someone declares a use case for it.

Comment (medyagh) [reply pkg/minikube/bootstrapper/kubeadm/kubeadm.go]: I think the use case for it is, if I am developing locally, I wanna know my developoing-code created this cluster not the minikube stable

**Pull request merged**